From 0627c187970f195e95f1a9de69a6037fad57c2cb Mon Sep 17 00:00:00 2001
From: Gao Pan <pandy.gao@nxp.com>
Date: Mon, 4 Dec 2017 13:25:51 +0800
Subject: [PATCH 2955/5242] MLK-17061-2 arm64: dts: add interrupt-open-drain
 property for sensors

commit  e9d181a5b3196ed6e19fdde14fc8fb293005eac9 from
https://source.codeaurora.org/external/imx/linux-imx.git

add interrupt-open-drain property for sensors

Signed-off-by: Gao Pan <pandy.gao@nxp.com>
(cherry-picked from 7b8cd1c5d94f4e2ea5462ca490da6e8f125c92cf)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
index 701859c..8352967 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
@@ -596,16 +596,19 @@
 			fxos8700@1e {
 				compatible = "fsl,fxos8700";
 				reg = <0x1e>;
+				interrupt-open-drain;
 			};
 
 			fxas2100x@21 {
 				compatible = "fsl,fxas2100x";
 				reg = <0x21>;
+				interrupt-open-drain;
 			};
 
 			mpl3115@60 {
 				compatible = "fsl,mpl3115";
 				reg = <0x60>;
+				interrupt-open-drain;
 			};
 		};
 
-- 
1.7.9.5

